Airbrakes/imu.py

```diff
@@ -83,17 +83,28 @@ def _fetch_data_loop(self, port: str, frequency: int, _: bool):
         """
         This is the loop that fetches data from the IMU. It runs in parallel with the main loop.
         """
-        while self.running.value:
-            # Connect to the IMU
-            connection = mscl.Connection.Serial(port)
-            node = mscl.InertialNode(connection)
```

Comment:
i'm not sure why we keep reconnecting to the imu every loop. Once should be enough isn't it?

Reply:
that should be in init yeah, no need to keep making serial objects

Reply:
sorry I must have put it in the wrong spot. It shouldn't be in init, it should be in the process but above the while loop